source/usage-examples/updateOne.txt

```diff
@@ -52,6 +52,25 @@ see the :manual:`MongoDB update operator reference documentation
 
 .. include:: /includes/connect-guide-note.rst
 
-.. literalinclude:: /code-snippets/usage-examples/updateOne.js
-  :language: javascript
-  :linenos:
+.. tabs::
+
+  .. tab:: JavaScript
+     :tabid: javascript
+
+     .. literalinclude:: /code-snippets/usage-examples/updateOne.js
+        :language: javascript
+        :linenos:
+
+  .. tab:: TypeScript
+     :tabid: typescript
+
+     .. literalinclude:: /code-snippets/usage-examples/updateOne.ts
+        :language: typescript
+        :linenos:
+
+If you run the example above, you should see output that resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   1 document(s) matched the filter, updated 0 document(s)
```

Comment:
This output is incorrect in my local testing against a fresh sample dataset

Reply:
Great catch. Apologies, I should have noticed this, that is odd output. I see that the code produces different outputs between the first and subsequent updates.

I changed the example so that it uses a random number to always update the document in the collection. I changed the document to a movie called "Random Harvest" that is in the sample collection.